app/views/tournaments/show_pools.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1><%= @tournament.name %></h1>
+  <% @pools.each do |pool| %>
+    <div>
```

Comment:
Should this be a `ul` within `ul`? A list element for each pool, plus another for each fighter.